src/client/proxy/handlers/storage.ts

```diff
@@ -8,21 +8,27 @@ import {Template, templateManager} from '../templateManager';
 // TODO: we don't handle multiple files upload. But if we want to,
 // we should change the response format
 const attachStorageHandlers = (server: FastifyInstance) => {
-    ['/_storage/', '/_storage'].forEach(route => {
-        server.post(route, async (req, res) => {
-            if (!req.headers['content-type']?.match('multipart/form-data')) {
-                return res.status(415).send('Only multipart/form-data is supported');
-            }
+    server.post('/_storage/', async (req: any, res: any) => {
```

Comment:
This change should be tested, as long as the path is used in many dapps like `social.point` and others. @jensendarren, could you please confirm it works like that?